src/main/java/interface_adapter/navigation/NavPresenter.java

```diff
@@ -0,0 +1,57 @@
+package interface_adapter.navigation;
+
+import interface_adapter.ViewManagerModel;
+import interface_adapter.change_password.LoggedInViewModel;
+import interface_adapter.login.LoginViewModel;
+import interface_adapter.recipe_collection.RecipeCollectionViewModel;
+import interface_adapter.search.SearchViewModel;
+import interface_adapter.signup.SignupViewModel;
+import use_case.navigation.NavOutputBoundary;
+
+public class NavPresenter implements NavOutputBoundary {
```

Comment:
A Javadoc here describing your NavPresenter will eliminate a Checkstyle error and improve clarity on what your presenter does!